src/burn/engine/registration.cpp

```diff
@@ -521,14 +517,6 @@ extern "C" HRESULT RegistrationDetectResumeType(
     HKEY hkRegistration = NULL;
     DWORD dwResume = 0;
 
-    if (IsBundleRebootPending(pRegistration))
-    {
-        LogId(REPORT_STANDARD, MSG_PENDING_REBOOT_DETECTED, pRegistration->sczRegistrationKey);
-
-        *pResumeType = BOOTSTRAPPER_RESUME_TYPE_REBOOT_PENDING;
```

Comment:
We should also remove `BOOTSTRAPPER_RESUME_TYPE_REBOOT_PENDING`.

Reply:
D'oh. Thought I caught that one.